tests/light/functional_tests/destination_drivers/network_destination/test_tls_verifier_reload_crash.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2024 Axoflow
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+from src.common.file import copy_shared_file
+
+
+def test_tls_verifier_reload_crash(config, syslog_ng, port_allocator, testcase_parameters):
```

Comment:
I think the testcase name will not relevant after this fix. So I would suggest other name (or any other similar to this): 
`test_send_log_for_tls_destination_after_a_reload()`

Reply:
I think tests that were created for specific bugs should capture the fact in their name, otherwise, one could refactor this test without knowing its purpose (for example, sending more messages before reloading will hide the complex bug we fixed)

Reply:
Renamed the function and the file to `test_kept_alive_tls_connection_doing_handshake_after_reload`.

Reply:
Thanks, I like this new testcase name.